src/senselab/audio/data_structures/audio.py

```diff
@@ -311,7 +313,7 @@ def save_to_file(
             if not os.path.exists(output_dir):
                 os.makedirs(output_dir)
             torchaudio.save(
-                uri=file_path,
+                uri=str(file_path),
```

Comment:
there a reason for this conversion? file_path should already be PathLike (or string)